### ISSUE TITLE
armv7-unknown-linux-gnueabihf bindings

### ISSUE THREAD

**Issue by AlexanderGuncenko** [DelSkayn/rquickjs#95]
Hello 👋 ,
can you please add armv7-unknown-linux-gnueabihf bindings .

**Comment by katyo** [comment]
@AlexanderGuncenko Currently we have some problems related to using `bindgen` with cross compilation.

**Comment by Sytten** [comment]
Yeah this is an issue, personally giving it a try with cross and custom images
